entry_tasks/77_Remove_each_second/77_Remove_each_second.py

```diff
@@ -0,0 +1,8 @@
+if __name__ == '__main__':
+    nums = input("Please enter  sequence of numbers : ").split(',')    #Example: 3,2,1,4,11,5,2
+    del nums[1::2]
+    for i in range(len(nums)):
+        nums[i] = str.strip((nums[i]))
+        nums1 = nums[::-1]
```

Comment:
I believe you didn't want to do this inside for loop, right?
Because you create reversed list on each step of the for loop.
But you need it only once, in the very end of your program, correct?